x/version/chains.go

```diff
@@ -16,10 +16,14 @@ func StandardChainVersions() map[string]ChainVersionConfig {
 	mainnetVersions := NewChainVersionConfig(map[uint64]int64{
 		1: 0,
 	})
+	bsr := NewChainVersionConfig(map[uint64]int64{
+		0: 0,
+		1: 674000,
+	})
```

Comment:
probably could have been a separate PR, but this includes the upgrade height for BSR